src/libraries/System.Resources.ResourceManager/tests/ResourceManagerTests.cs

```diff
@@ -82,6 +84,7 @@ public static IEnumerable<object[]> CultureResourceData()
 
         [Theory]
         [MemberData(nameof(CultureResourceData))]
+        [ActiveIssue("https://github.com/dotnet/runtimelab/issues/155", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
This issue has a new home: https://github.com/dotnet/runtime/issues/69919

It would be nice to update the existing references of https://github.com/dotnet/runtimelab/issues/155 to this one.

Reply:
We can do it in one go later. 155 is a number that I don't need to look up because I remember it. 69919 is harder to remember. I often need it during bringup and I want any tiny help that makes the bringup easier. It's very tedious and repetitive.